app/sign.js

```diff
@@ -131,12 +132,10 @@ const promptForConfirmationAndKey = function(recoveryRequest, outputs, skipConfi
  * Gets the backup private key that can be used to sign the transaction.
  * @param xprv The provided extended private key (BIP32).
  * @param expectedXpub The public key specified with the request.
- * @returns The private key to sign the transaction.
+ * @returns Buffer - the private key buffer to sign the transaction.
```

Comment:
Should this be `{Buffer}` to denote a type?

Reply:
intellij at least doesn't seem to need them